testdata/input.txt

```diff
@@ -76,3 +76,4 @@ Ben Hur 2016 TELESYNC x264 AC3 MAXPRO
 The.Secret.Life.of.Pets.2016.HDRiP.AAC-LC.x264-LEGi0N
 [HorribleSubs] Clockwork Planet - 10 [480p].mkv
 [HorribleSubs] Detective Conan - 862 [1080p].mkv
+Blade.Runner.2049.2017.1080p.WEB-DL.DD5.1.H264-FGT-[rarbg.to]
```

Comment:
I did't check it out but you should also check the `2012` movie. Is it working in this cases?

1. `2012(2009).1080p.Dual Audio(Hindi+English) 5.1 Audios`
2. `2012 (2009) 1080p BrRip x264 - 1.7GB - YIFY`
3. `2012 2009 x264 720p Esub BluRay 6.0 Dual Audio English Hindi GOPISAHI`